### PR TITLE
Updating firewall settings and linking to relevant sections - Closes#522

### PR DESCRIPTION
**Pull request by asettle**
Also fixes syntax for "system" tag

### CODE REVIEW

File: xml/admin_gui_dashboard.xml [hunk 1]
```diff
@@ -2676,12 +2682,12 @@ Please consult the documentation on how to configure and enable the Object Gatew
    <title>Enabling the &ogw; Management Front-end</title>
    <para>
     To use the &ogw; management functionality of the dashboard, you need to
-    provide the login credentials of a user with the 'system' flag enabled:
+    provide the login credentials of a user with the <option>system</option> flag enabled:
```
Comment (LenzGr) [inline xml/admin_gui_dashboard.xml:2685]: This and the following change don't seem to be related to this PR?

Comment (asettle) [reply xml/admin_gui_dashboard.xml:2685]: Yeah that's just me adding in syntax :eyes:

File: xml/admin_gui_dashboard.xml [hunk 2]
```diff
@@ -2621,6 +2621,12 @@ Please consult the documentation on how to configure and enable the Object Gatew
     port. By default, the currently active &mgr; that hosts the dashboard binds
     to TCP port 8443 (or 8080 when SSL is disabled).
    </para>
+   <note>
+     <para>
+       For more information on firewall settings for &ceph;, see
```
Comment (LenzGr) [inline xml/admin_gui_dashboard.xml:2626]: I would lead into that note with a hint about "If a firewall is enabled on the hosts running Ceph Mgr (and thus Ceph Dashboard), you may have to change it's configuration to enable access to these ports."

File: xml/admin_gui_dashboard.xml (hunk 1 above)
Comment (asettle) [reply xml/admin_gui_dashboard.xml:2685]: "when in Rome"

File: xml/admin_gui_dashboard.xml (hunk 2 above)
Comment (asettle) [reply xml/admin_gui_dashboard.xml:2626]: Verbosity is my velocity

File: xml/admin_gui_dashboard.xml [hunk 3]
```diff
@@ -2710,6 +2718,12 @@ Please consult the documentation on how to configure and enable the Object Gatew
 </screen>
     </step>
    </procedure>
+   <note>
+     <para>
+       By default the firewall is enabled with SLES15SP2, for more information
```
Comment (lproven) [inline xml/admin_gui_dashboard.xml:2723]:
```suggestion
       By default the firewall is enabled in &sls; 15 SP2. For more information
```

**Review by lproven — CHANGES_REQUESTED** [review]

**Review by lproven — APPROVED** [review]

**Comment by asettle** [comment]
@LenzGr if you're happy, I'll merge

**Review by LenzGr — APPROVED** [review]
I'm happy. Thank you!
